libpam/include/security/_pam_types.h

```diff
@@ -166,6 +166,12 @@ typedef struct pam_handle pam_handle_t;
 # define PAM_DEPRECATED
 #endif
 
+#if PAM_GNUC_PREREQ(2,7)
+# define PAM_UNUSED __attribute__((__unused__))
+#else
+# define PAM_UNUSED
+#endif
```

Comment:
Given that m4/attribute.m4 already defines `UNUSED`, does it worth exporting a new macro just for use in PAM_DEBUG case?

Wouldn't it be sufficient to use
```
#ifdef UNUSED
UNUSED
#endif
```
in _pam_macros.h?